validator/sawtooth_validator/execution/scheduler_parallel.py

```diff
@@ -96,6 +96,15 @@ def get(self, address):
 
         return node
 
+    def get_nodes_above(self, address):
```

Comment:
A docstring would be helpful here, to clarify what `above` means

Reply:
Renamed to `get_ancestors` and docstring added